addons/PieChart/src/presenter.js

```diff
@@ -256,7 +256,7 @@ function AddonPieChart_create(){
             x1 = parseFloat(presenter.center + presenter.radius * Math.sin(angle));
             y1 = parseFloat(presenter.center - presenter.radius * Math.cos(angle));
             angle2 = angle + 0.5*(parameter)/50 * Math.PI;
-            angle += (parameter)/50 * Math.PI;
+            angle += (parameter)/50 * Math.PI; //if this changes, fix generateItemData
```

Comment:
Komentarze bywają problematyczne. 
Przez nieuwagę kolejnej osoby mogą zostać skasowane.
Lub przy poprawie presenter.generateItemData nie zostaną poprawione tutaj rzeczy.
Lub sam komentarz może się odnosić do większej liczby linijek i przy kolejnych zmianach konieczna będzie analiza co czego konkretnie odwołuje się komentarz.
Lub kiedyś komentarz może się okazać zbędny "ale się go zostawi tak na wszelki wypadek, bo może...". 

Lepszym rozwiązaniem jest wydzielenie wspólnej części do osobnej metody. 
W ten sposób minimalizujemy takie przypadki + kasujemy powtarzające się fragmenty kodu.
Jeżeli nie da się wydzielić większego fragmentu to można z mniejszym fragmentem.